hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/store/driver/StateStoreDriver.java

```diff
@@ -35,6 +36,7 @@
  * provider. Driver implementations will extend this class and implement some of
  * the default methods.
  */
+@InterfaceAudience.Public
```

Comment:
Let's also add `@InterfaceStability.Evolving`?

Reply:
Sounds good, since it is not yet stable.